RMSD.py

```diff
@@ -113,6 +123,11 @@ def RMSD(ref_mol2, query_mol2):
     # set up a log file for RMSD
     logFile = open('%s/%s' % (directory, opt.output) ,'a')
 
+    has_outlier = False
```

Comment:
Hmm, so the variable name sounds like it checks whether or not this set has an outlier. But reviewing the code makes it seem like the variable represents whether or not to check for outliers. Is that interpretation correct?

Reply:
So I have two variable, has_outlier I was thinking of choosing to store outliers or not, maybe a better name for this would be storing_outliers?

Reply:
yeah, I think either storing_outliers or store_outliers would be more readable.